common/src/main/scala/hmda/serialization/submission/SubmissionSerializer.scala

```diff
@@ -0,0 +1,35 @@
+package hmda.serialization.submission
+
+import java.io.NotSerializableException
+
+import akka.serialization.SerializerWithStringManifest
+import hmda.model.filing.submission.Submission
+import hmda.persistence.serialization.submission.SubmissionMessage
+import hmda.serialization.submission.SubmissionProtobufConverter._
+
+class SubmissionSerializer extends SerializerWithStringManifest {
```

Comment:
This class is not necessary. The `SubmissionCommandsSerializer` already has support for serializing `Submission` objects. All that is required is the following line in the `serialization.conf` file: 

`hmda.model.filing.submission = submission-commands`